tests/e2e/package.json

```diff
@@ -5,8 +5,9 @@
   "main": "dist/index.js",
   "scripts": {
     "lint": "tslint --fix -p .",
-    "tsc": "./configs/sh-scripts/generateIndex.sh && tsc -p .",
+    "tsc": "rm -rf ./dist && ./configs/sh-scripts/generateIndex.sh && tsc -p .",
     "test": "./configs/sh-scripts/initDefaultValues.sh npm run lint && npm run tsc && export USERSTORY=$USERSTORY && mocha --config dist/configs/mocharc.js",
+    "test-api": "export TS_USE_WEB_DRIVER_FOR_TEST=false && npm run test",
```

Comment:
`driver-less-test` command name is more relevant in here, IMHO

Reply:
no problem, renamed